src/sweeps/bayes_search.py

```diff
@@ -302,7 +299,18 @@ def next_sample(
         best_test_X_index = np.argmax(prob_of_improve)
     else:
     """
-    best_test_X_index = np.argmax(e_i)
+
+    # Check to see that the metric varies accross the samples
+    if np.all(np.isclose(filtered_y, filtered_y[0], atol=METRIC_VALUE_ATOL)):
+        logging.warning(
+            f"All instances of metric are within the minimum tolerance of {METRIC_VALUE_ATOL},"
+            + "the next sample will be a random sample within parameter space"
+        )
```

Comment:
this will warn into the server logs which the user will never see. we should route warnings back to users